sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/core/PropertiesValidator.java

```diff
@@ -21,25 +21,25 @@ public static void validateNamespace(String namespace) {
 
     private static void validateLength(String namespace) {
         if (namespace.length() < 6 || namespace.length() > 50) {
-            throw LOGGER.logExceptionAsError(new IllegalArgumentException(LENGTH_ERROR));
+            LOGGER.warning(LENGTH_ERROR);
```

Comment:
We should not use the ClientLogger here.

Reply:
How about we still throw the exception here, and in the configuration properties, we catch the exception and logger it. Printing warning message is not what a validator should do.